benchmark/README.md

```diff
@@ -119,6 +119,9 @@ installed libraries instead, enable
 This will reflect the performance of the Swift standard library
 installed on the device, not the one included in the Swift root.
 
+If a build fails with an error like `cannot find X in scope`, it may be because
+default toolchains don't give access to stdlib SPI. Try using a dev toolchain
+built locally or a dev snapshot from [Swift.org](https://www.swift.org/install/).
```

Comment:
This line is still longer.

Reply:
I thought it would mess up rendering. But I was wrong. Will update.